### ISSUE TITLE
Feature request: Alien::hunspell

### ISSUE THREAD

**Issue by run4flat** [cosimo/perl5-text-hunspell#3]
Makefil.PL couldn't detect libhunspell on my Mac, even after installing hunspell with homebrew. That's as far as I'm willing to take my efforts when trying to install a module, and I'm sure that's more effort than most. The best solution, I think, would be to have Alien::hunspell, which could install a Perl-local copy of hunspell if it could not link against the local machine's hunspell.

I don't have tuits for this at the moment, but would be happy to help out in a couple of months if you're interested.

**Comment by plicease** [comment]
We (@Perl5-Alien) we just discussing this and it only took me a few minutes to create an `Alien::Base` based `Alien::Hunspell`:

https://github.com/plicease/Alien-Hunspell

I think the challenge is getting an appropriate word list though.

cc: @zmughal

**Comment by plicease** [comment]
Also, one disadvantage of `ExtUtils::PkgConfig` is that it won't install unless you have `pkg-config`, whereas `Alien::Hunspell` will install on systems without `pkg-config`.

**Comment by cosimo** [comment]
I don't know what it would mean to integrate Alien::Hunspell and if that solution is more compatible/portable across different OSes. I tried looking at the documentation here (https://metacpan.org/pod/Alien::Hunspell) but I couldn't find anything that indicates how to use it.

What would happen on Debian-derivative systems where a libhunspell package is present?

**Comment by plicease** [comment]
I apologize the documentation for `Alien::Hunspell` is sparse!  But it basically uses the same interface as all `Alien::Base` based distros.  I've opened a PR which makes the changes for you: #8 

As for Linux vendors, I'm available to work with distribution developers to make sure that AB based distros work with their build tools.  I've worked specifically with @leonerd with to make sure that `Alien::Sodium` (also AB based) works with the Debian tools, and I believe he has some preliminary working .deb files.

I recognize that there are tradeoffs here.  The potential complication for system integrators vs. the convenience of more casual CPAN users.  However, once the AB dependency is solved for a particular Linux vendor, supporting other AB based dependencies is relatively light, compared to non AB based distributions which each have their own way of doing things.
